CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 24.3.1
+
+New error codes:
+* Y064: Use simpler syntax to define final literal types.
+  For example, use `x: Final = 42` instead of `x: Final[Literal[42]]`
```

Comment:
#469 didn't have a changelog entry

Reply:
I guess this was already part of 24.3.0? In that case, I'd just add this entry there retroactively, and maybe add a comment that the changelog entry was missing to the latest version.

Reply:
No, it hasn't been included in any release yet